addon/components/frost-popover.js

```diff
@@ -515,12 +515,14 @@ export default Component.extend(PropTypeMixin, {
 
       if (this.get('visible')) {
         this.registerClickOff()
-        let offset = this.resolvePlacement()
 
+        const $element = $(this.get('element'))
+        $element.css('display', 'block')
+        let offset = this.resolvePlacement()
         const delta = this.getViewportAdjustments(position)
 
         // apply delta to contain the popover inside the viewport
-        $(this.get('element')).css({
+        $element.css({
```

Comment:
This logic is only relying on being lucky with rendering timing, right?  My first thought was to write

> combine the style declarations into single call to `.css()`

but then I started thinking that the placement must need to first be resolved (`this.resolvePlacement()`) before being able to set the other properties.   So then I looked at what is going on and this comment - https://github.com/twjohnston/ember-frost-popover/blob/8dc3415d04af8148c6e74645ab09d05944a6ebc4/addon/components/frost-popover.js#L474-L476 - and the code, oh boy.  There is a whole lot of stuff going on in a separate thread (UI rendering vs javascript execution) that has no meaningful way to tying the timing back together between the two.   We have been getting lucky with this code up till now and are relying on continuing to get lucky.

Having said all that I will approve this PR unless you know of a quick fix to address this then please implement it.

Reply:
There are definitely some timing issues at play here. I don't have a quick fix at the moment, but it's something that should be revisited for sure.

Reply:
I have opened https://github.com/ciena-frost/ember-frost-popover/issues/80